src/Themes.scss

```diff
@@ -242,7 +242,7 @@ body.light-theme {
 
     button.btn.AddNode__expanded-button {
         background-color: white;
-        border: 3px solid #505862;
+        border: 2px solid #505862;
```

Comment:
Why not use the `addNodeExpandedButtonBorderWidth` variable here?

Reply:
Currently, addNodeExpandedButtonBorderWidth is defined in AddNode.scss and from the theming work, any border for the expanded add node button are hardcoded in Theme.scss.

Reply:
I am thinking, we should merge this PR with this change, and have a clean up task to move any commonly used values in different themes to variables.scss, what do you think?

Reply:
That sounds reasonable.  We could also consider making some more animations into mixins.

Reply:
This work is ready for a review @sbates-idrc, please let me know what do you think about the comments Tony and I left.